app/templates/components/global-search.hbs

```diff
@@ -2,11 +2,11 @@
   type="search"
   placeholder={{t 'general.globalSearchPlaceholder'}}
   incremental=true
-  onsearch={{action "inputValueChanged"}}
-  onkeyup={{action "inputValueChanged"}}
+  oninput={{action "inputValueChanged"}}
```

Comment:
onsearch event is much better and onkeyup provides cross browser compatibility.  Please revert.